packages/draft-js-import-element/src/stateFromElement.js

```diff
@@ -298,7 +308,17 @@ class ContentGenerator {
     let block = this.blockStack.slice(-1)[0];
     let style = block.styleStack.slice(-1)[0];
     let entityKey = block.entityStack.slice(-1)[0];
-    style = addStyleFromTagName(style, tagName, this.options.elementStyles);
+    let {customInlineFn} = this.options;
+    let customInline = customInlineFn ? customInlineFn(element) : null;
+    if (typeof customInline === 'string') {
+      style = style.add(customInline);
+    } else if (customInline != null) {
+      // TODO: do something with the block data
```

Comment:
No, a [Content Block](https://draftjs.org/docs/api-reference-content-block.html) and [Entity](https://draftjs.org/docs/advanced-topics-entities.html) are different in DraftJS. A block is like a paragraph. An entity is a range of text inside a block that has special meaning/data attached to it. Like `<a href="...">foo</a>`.

I'll try to show you an example of what I mean.